diff --git a/node_modules/eth-trezor-keyring/index.js b/node_modules/eth-trezor-keyring/index.js
index 9b76a29..d27ca88 100644
--- a/node_modules/eth-trezor-keyring/index.js
+++ b/node_modules/eth-trezor-keyring/index.js
@@ -5,7 +5,7 @@ const TrezorConnect = require('trezor-connect').default;
 const { TransactionFactory } = require('@ethereumjs/tx');
 const transformTypedData = require('trezor-connect/lib/plugins/ethereum/typedData');
 
-const hdPathString = `m/44'/60'/0'/0`;
+const hdPathString = `m/44'/1642'/0'/0`;
 const SLIP0044TestnetPath = `m/44'/1'/0'/0`;
 
 const ALLOWED_HD_PATHS = {
